### PR TITLE
Bonus: Add shell status script

### PR DESCRIPTION
**Pull request by dennisreimann**
Adds an additional "Bonus" section to the parent deployment section.

bonus

### CODE REVIEW

**Comment by pavlenex** [comment]
Not found on this being in the structure sidebar, bonus sounds weird to me and I don't think it adds good navigation since this is the only choice. I'd add this as a FAQ question or something with themeing customization although not entirely related to that, but certainly not as page. I don't think we should clutter docs with these optional stuff.

**Comment by dennisreimann** [comment]
Fair enough.

**Comment by Zaxounette** [comment]
Simple though:

Instead of `Bonus`, the title in the nav bar could be `Miscellaneous`
Would sound less "weird".

File: docs/Bonus/StatusScript.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+sudo chmod +x /usr/local/bin/btcpay-status
+
+# check if it works
+sudo btcpay-status
+
+# if so, you can also set it up as a welcome script
+sudo ln -s /usr/local/bin/btcpay-status /etc/update-motd.d/btcpay-status
+```
+
+You might need to install the dependeny `jq`, which is a prerequisite:
```
Comment (britttttk) [inline docs/Bonus/StatusScript.md:25]: dependency

**Comment by britttttk** [comment]
I agree with @pavlenex I think this would be better suited as an FAQ and not in the nav bar.

**Comment by dennisreimann** [comment]
@Kukks will integrate this directly into the docker deployment, so we don't have to have this page at all 😀

**Pull request closed without merging**